src/views/product-integrations-landing/components/searchable-integrations-list/style.module.css

```diff
@@ -26,9 +26,12 @@
 	display: flex;
 	flex-wrap: wrap;
 	gap: 12px;
+	min-height: 30px;
```

Comment:
Fixed some annoying layout shift when changing between "no filters" and "some filters"